packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -19,16 +39,16 @@ export interface ChartsTooltipProps extends Omit<ChartsTooltipContainerProps, 'c
  * - [ChartsTooltip API](https://mui.com/x/api/charts/charts-tool-tip/)
  */
 function ChartsTooltip(props: ChartsTooltipProps) {
-  const { classes: propClasses, trigger = 'axis' } = props;
+  const { classes: propClasses, trigger = 'axis', slots, slotProps } = props;
 
   const classes = useUtilityClasses(propClasses);
 
   return (
-    <ChartsTooltipContainer {...props} classes={classes}>
+    <ChartsTooltipContainer slots={slots} slotProps={slotProps} {...props} classes={classes}>
```

Comment:
```suggestion
    <ChartsTooltipContainer {...props} classes={classes}>
```

Or if you do `slots, slotProps, ...other } = props;`

```suggestion
    <ChartsTooltipContainer slots={slots} slotProps={slotProps} classes={classes} {...other}>
```